scripts/test-ci.sh

```diff
@@ -7,11 +7,10 @@ npm test
 
 # we need to build to run functional tests
 NODE_ENV=production npm run build
+rm -rf functional-tests/screenshots
```

Comment:
Maybe we could also ensure the folder exists here or just configure some module to create it automatically and empty it.